### ISSUE TITLE
Sync tomahawh-player/tomahawk with  lp:~tomahawk/tomahawk/master does not work

### ISSUE THREAD

**Issue by justin-time** [tomahawk-player/tomahawk#357]
The sync between github and launchpad is broken. And so the ubuntu nightlys will not be updated any longer!

Last nightly: 31.Oct.2015!

**Comment by muesli** [comment]
On my todo-list, we had to migrate to new servers due to hardware failure in November. Sorry for that!

**Comment by justin-time** [comment]
No problem, it looks like neither new windows nightly build nor mac build exists. And so its ok that ubuntu builds do not work, too. :-)

**Comment by dschmidt** [comment]
Yes, certainly related. I'm optimistic we're going to have OSX builds early next week again.
